package.json

```diff
@@ -16,7 +17,8 @@
   },
   "homepage": "https://github.com/dsb-norge/vue-keycloak-js",
   "dependencies": {
-    "keycloak-js": "9.0.3"
+    "keycloak-js": "9.0.3",
+    "vue-demi": "latest"
```

Comment:
Not quite sure that we need to introduce a new dependency for this support. What you had earlier worked fine in a Vue 2 project given the little check i mentioned in the other comment. While with the current way it crashes when attempting to reference "vue.reactive" when the package is built. 

Haven't tried for a Vue 3 project though, as i assume thats what you are testing it with. As mentioned in https://github.com/dsb-norge/vue-keycloak-js/issues/56 we have not yet started to use Vue 3.